packages/db-migrations/databases/fxa/patches/patch-131-132.sql

```diff
@@ -0,0 +1,120 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+-- Make sure there are meta data settings for PrunedUntil and LastPrunedAt
+INSERT IGNORE INTO dbMetadata (name, value) VALUES ('sessionTokensPrunedUntil', '0');
```

Comment:
nit: I think it might be clearer to rename this as `prunedUntil` `lastPrunedAt` since it does more than just sessionTokens.

Reply:
Agreed.